test/check-application

```diff
@@ -1451,7 +1468,19 @@ class TestFiles(testlib.MachineCase):
         self.enter_files()
 
         # Check control-clicking
-        m.execute("touch /home/admin/file1 && touch /home/admin/file2")
+        dir_path = "/home/admin"
```

Comment:
nitpick, you don't really re-use dir_path anywhere below so you can just drop it